docs/usage/configuration-options.md

```diff
@@ -2375,7 +2375,9 @@ You can configure the `rollbackPrs` property globally, per-lanuage, or per-packa
 ## schedule
 
 The `schedule` option allows you to define times of week or month for Renovate updates.
-Running Renovate around the clock may seem too "noisy" for some projects and therefore `schedule` is a good way to reduce the noise by reducing the timeframe in which Renovate will operate on your repository.
+Running Renovate around the clock can be too "noisy" for some projects.
+To reduce the noise you can use the `schedule` config option to limit the time frame in which Renovate will run on your repository.
```

Comment:
```suggestion
To reduce the noise you can use the `schedule` config option to limit the time frame in which Renovate will perform actions on your repository.
```

Renovate will always run on the repo, but it may not e.g. create branches